x-pack/plugins/ml/public/application/jobs/new_job/pages/components/common/json_editor_flyout/json_editor_flyout.tsx

```diff
@@ -110,19 +145,51 @@ export const JsonEditorFlyout: FC<Props> = ({ isDisabled, jobEditorMode, datafee
                     defaultMessage: 'Job configuration JSON',
                   })}
                   value={jobConfigString}
+                  heightOffset={showChangedIndicesWarning ? WARNING_CALLOUT_OFFSET : 0}
```

Comment:
I presume the callout height is not static and depends on the window width. So probably you should use the reference to the element and check the actual value.